src/components/zine-card.tsx

```diff
@@ -24,7 +24,7 @@ const ZineCard: React.FC<ZineCardProps> = ({ zine }) => {
         <div className="flex flex-col mt-3 text-center">
           <h1 className="text-lg font-medium">
             {zine.title}{" "}
-            <span className="text-gray-500 text-sm">por {zine.author_name ?? ""}</span>
+            <span className="text-gray-500 text-sm">por {zine.library_zines_authors.map((a) => a.authors.name).join(", ")}</span>
```

Comment:
O que acha de fazer uma função helper pra isso? Bastante repetido esse padrão.

```suggestion
            <span className="text-gray-500 text-sm">por {joinAuthors(zine.library_zines_authors)}</span>
```

Reply:
boa, função criada e aplicada